util/gitutil/gitutil.go

```diff
@@ -116,6 +117,9 @@ func (c *Git) run(args ...string) (string, error) {
 		cmd.Dir = c.wd
 	}
 
+	// Override the locale to ensure consistent output
+	cmd.Env = append(os.Environ(), "LC_ALL=C")
```

Comment:
Before I facepalm myself for not asking; did someone look (or do we need a tracking ticket) on the BuildKit side if we should also set this option there? (I think BuildKit also shells out to `git` - it may be in a more predictable environment, but possibly could still be a good thing to do)

Reply:
It might be useful on BuildKit too so we have a consistent output when users report issues.